README.md

```diff
@@ -1,6 +1,106 @@
 # sig-clients
 
+This is the repository for the Clients Special Interest Group (sig-clients) in
+the Sigstore project. This group has the following provisional mission until the
+next meeting where we'll discuss it:
+
+> Make Sigstore clients across languages/ecosystems easy-to-write, compatible,
+> and secure by providing shared designs/documentation, data formats, and test
+> suites.
+
+In general, we'll try to avoid telling individual implementations what to do,
+though we may have criteria for various official statuses (e.g., what
+constitutes a "supported" client).
+
+## Projects
+
+sig-clients doesn't exclusively own most of these, but you may be interested:
+
+- [protobuf-specs](https://github.com/sigstore/protobuf-specs): cross-client data formats, especially for serializing 
+- [sigstore-conformance](https://github.com/sigstore/sigstore-conformance): conformance test suites for clients
+- [architecture-docs](https://github.com/sigstore/architecture-docs]): specifications
+- Sigstore client implementations in various languages:
+  - Go: [sigstore-go](https://github.com/sigstore/sigstore-go), [Cosign](https://github.com/sigstore/cosign), [Gitsign](https://github.com/sigstore/gitsign),
+  - Java: [sigstore-java](https://github.com/sigstore/sigstore-java)
+  - JavaScript: [sigstore-js](https://github.com/sigstore/sigstore-js)
+  - Python: [sigstore-python](https://github.com/sigstore/sigstore-python)
+  - Ruby: [sigstore-ruby](https://github.com/sigstore/sigstore-ruby)
+  - Rust: [sigstore-rs](https://github.com/sigstore/sigstore-rs)
+[policy-controller](https://github.com/sigstore/policy-controller)
```

Comment:
Is policy controller in scope? This seems like more than just a sigstore client

Reply:
> more than just a sigstore client

So is Cosign :)

I think "resolve discrepancies between policy-controller and Cosign" is *plausibly* in-scope, and relates to a lot of the near-term UX/policy issues. Given that we're not asserting any authority here over the project, just saying that policy-controller might care about some of the decisions we make, I'm comfortable erring on the side of listing more projects.